[2024-08-01T15:15:36.928+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-01T15:15:36.962+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fill_tables_on_rd.Truncate_product manual__2024-08-01T15:15:26.786282+00:00 [queued]>
[2024-08-01T15:15:36.971+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fill_tables_on_rd.Truncate_product manual__2024-08-01T15:15:26.786282+00:00 [queued]>
[2024-08-01T15:15:36.972+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-08-01T15:15:36.987+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): Truncate_product> on 2024-08-01 15:15:26.786282+00:00
[2024-08-01T15:15:36.994+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'fill_tables_on_rd', 'Truncate_product', 'manual__2024-08-01T15:15:26.786282+00:00', '--job-id', '740', '--raw', '--subdir', 'DAGS_FOLDER/fill_tables.py', '--cfg-path', '/tmp/tmp1dm3n5vg']
[2024-08-01T15:15:36.997+0000] {standard_task_runner.py:91} INFO - Job 740: Subtask Truncate_product
[2024-08-01T15:15:36.997+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=2218) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-08-01T15:15:36.998+0000] {standard_task_runner.py:63} INFO - Started process 2220 to run task
[2024-08-01T15:15:37.047+0000] {task_command.py:426} INFO - Running <TaskInstance: fill_tables_on_rd.Truncate_product manual__2024-08-01T15:15:26.786282+00:00 [running]> on host 2634975b7554
[2024-08-01T15:15:37.127+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='fill_tables_on_rd' AIRFLOW_CTX_TASK_ID='Truncate_product' AIRFLOW_CTX_EXECUTION_DATE='2024-08-01T15:15:26.786282+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-01T15:15:26.786282+00:00'
[2024-08-01T15:15:37.128+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-01T15:15:37.147+0000] {base.py:84} INFO - Using connection ID 'postgres_default2' for task execution.
[2024-08-01T15:15:37.168+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-08-01T15:15:37.170+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-01T15:15:37.179+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=fill_tables_on_rd, task_id=Truncate_product, run_id=manual__2024-08-01T15:15:26.786282+00:00, execution_date=20240801T151526, start_date=20240801T151536, end_date=20240801T151537
[2024-08-01T15:15:37.213+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-08-01T15:15:37.237+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-08-01T15:15:37.241+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
